### PR TITLE
Bump Hadoop containers to Apache Hadoop 2.7 (resolves #256)

### LINKED ISSUE #256: Bump to Apache Hadoop 2.7

### PR DESCRIPTION
**Pull request by fnothaft**

### CODE REVIEW

**Comment by fnothaft** [comment]
This is good to merge from my side. Ping @jvivian for a review/merge.

File: apache-hadoop-master/Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM quay.io/ucsc_cgl/apache-hadoop-common:2.6.2
+FROM quay.io/ucsc_cgl/apache-hadoop-common
```
Comment (jvivian) [inline apache-hadoop-master/Dockerfile:1]: Don't want to pin a version to pull from?

File: apache-hadoop-worker/Dockerfile [hunk 2]
```diff
@@ -1,4 +1,4 @@
-FROM quay.io/ucsc_cgl/apache-hadoop-common:2.6.2
+FROM quay.io/ucsc_cgl/apache-hadoop-common
```
Comment (jvivian) [inline apache-hadoop-worker/Dockerfile:1]: ditto

File: apache-hadoop-master/Dockerfile (hunk 1 above)
Comment (fnothaft) [reply apache-hadoop-master/Dockerfile:1]: Nope; look at the makefile. We build the latest `quay.io/ucsc_cgl/apache-hadoop-common` container prior to building this container. This makes it easier to update the two containers in tandem.

**Review by jvivian — APPROVED** [review]

**Pull request merged**